examples/bri-3/src/bri/workgroup/worksteps/agents/worksteps.agent.ts

```diff
@@ -57,6 +65,21 @@ export class WorkstepAgent {
     workstepToUpdate.updatePrivacyPolicy(privacyPolicy);
   }
 
+  public throwIfCircuitInputTranslationSchemaInvalid(schema): void {
+    const [result, error] =
+      this.cips.validateCircuitInputTranslationSchema(schema);
+    if (!result) {
```

Comment:
probably should check if there is error instead of result and returning error imo